src/components/Signup.vue

```diff
@@ -59,6 +59,10 @@
                         v-model="newUser.confirm"
                         required
                 >
+              <br>
+              <div v-if="newUser.password == '' && newUser.confirm == ''"></div>
```

Comment:
AngularJS has a thing called clean and dirty fields. Does Vue have something similar. I would suggest using that as with the way it is currently you can pass in no password...